hplsql/src/main/java/org/apache/hive/hplsql/objects/TableClass.java

```diff
@@ -96,9 +96,9 @@ public class TableClass implements HplClass {
       } else { // single column
         Row row = new Row();
         row.addColumn(
-                self.hplClass().columns().get(0).columnName(),
-                self.hplClass().columns.get(0).columnTypeString(),
-                args.get(1));
```

Comment:
Seems to be an unrelated change.

Reply:
Actually there is no change in this class. As a part of initial solution some changes were in this class later they are reverted. It looks this change some format change.